src/view/deploy.view.ts

```diff
@@ -99,7 +99,7 @@ export class DeployView {
       this.tasklist.addTask({
         title: `Deploying ${entry.name}`,
         // eslint-disable-next-line consistent-return
-        task: async (ctx, task) => {
+        task: async (_ctx, task) => {
```

Comment:
Why this, if not used a simple `_` is also good
```suggestion
        task: async (_, task) => {
```